### PR TITLE
sam_bitwise_flag_filter - fix indent

### PR DESCRIPTION
**Pull request by lecorguille**
This MR should fix this issue:

```
File "/shared/ifbstor1/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/devteam/sam_bitwise_flag_filter/0b2424a404d9/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py", line 111
infile = open ( options.input_sam, 'r')
^
TabError: inconsistent use of tabs and spaces in indentation
```

### CODE REVIEW

File: tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.xml [hunk 1]
```diff
@@ -1,15 +1,17 @@
-<tool id="sam_bw_filter" name="Filter SAM" version="1.0.0">
+<tool id="sam_bw_filter" name="Filter SAM" version="1.0.0+galaxy1">
```
Comment (nsoranzo) [inline tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.xml:1]: We don't need to track an upstream version here.
```suggestion
<tool id="sam_bw_filter" name="Filter SAM" version="1.0.1">
```

File: tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.xml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   <description>on bitwise flag values</description>
-  <parallelism method="basic"></parallelism>
-  <command interpreter="python">
-    sam_bitwise_flag_filter.py  
-      --input_sam_file=$input1
+  <requirements>
+    <requirement type="package" version="2.7">python</requirement>
+  </requirements>
+  <command detect_errors="aggressive"><![CDATA[
+    python $__tool_directory__/sam_bitwise_flag_filter.py
```
Comment (nsoranzo) [inline tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.xml:7]:
```suggestion
python '$__tool_directory__/sam_bitwise_flag_filter.py'
```

**Review by nsoranzo — COMMENTED** [review]
Can you please remove it from `.tt_skip` so it gets tested?

**Comment by wm75** [comment]
Weird, just out of interest I ran the tool on usegalaxy.org now with a BAM file as input.
It fails with:
```
Traceback (most recent call last):
  File "/cvmfs/main.galaxyproject.org/migrated_tools/toolshed.g2.bx.psu.edu/repos/devteam/sam_bitwise_flag_filter/0b2424a404d9/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py", line 148, in <module>
    if __name__ == "__main__": main()
  File "/cvmfs/main.galaxyproject.org/migrated_tools/toolshed.g2.bx.psu.edu/repos/devteam/sam_bitwise_flag_filter/0b2424a404d9/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py", line 137, in main
    flags = int( fields[flag_col] )
IndexError: list index out of range
```
there, but that's past the indent problem I guess?

File: tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py [hunk 3]
```diff
@@ -1,117 +1,118 @@
...
+import sys
 
-def stop_err( msg ):
-    sys.stderr.write( msg )
+
+def stop_err(msg):
+    sys.stderr.write(msg)
     sys.exit()
 
+
```
Comment (nsoranzo) [inline tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py:11]:
```suggestion
```

**Comment by nsoranzo** [comment]
The error in the CI logs doesn't make much sense to me, but is reproducible.

@mvdbeek Any ideas? Maybe the pip cache is busted?

**Comment by mvdbeek** [comment]
This might be fixed in https://github.com/galaxyproject/planemo/pull/1145. I'll push a commit that will install planemo from that branch

**Comment by mvdbeek** [comment]
Arg, even with the new release:
```
Activating virtualenv at /home/runner/.planemo/gx_venv_3.7_release_20.09
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
planemo 0.74.3 requires bioblend>=0.14.0, but you have bioblend 0.13.0 which is incompatible.
planemo 0.74.3 requires gxformat2>=0.12.0, but you have gxformat2 0.11.3 which is incompatible.
```

File: tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py [hunk 4]
```diff
@@ -1,117 +1,113 @@
...
         dest='flag_col',
-        default = '2',
+        default='2',
         help='Column containing SAM bitwise flag. 1-based')
 
     options, args = parser.parse_args()
 
     if options.input_sam:
-		infile = open ( options.input_sam, 'r')
+        infile = open(options.input_sam, 'r')
```
Comment (bernt-matthias) [inline tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py:107]: Can we close this file handle?

**Review by bernt-matthias — APPROVED** [review]
Looks good to me

**Comment by bernt-matthias** [comment]
And .. can you check if there is something in https://github.com/galaxyproject/tools-devteam/pull/555 that can be used?

File: tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py (hunk 4 above)
Comment (lecorguille) [reply tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py:108]: Done

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**